File: introduction/ITP1_3_B-Print Test Cases.js
process.stdin.resume();
process.stdin.setEncoding('utf8');
process.stdin.on('data', function (chunk) {
  var lines = chunk.toString().split('\n');
  for (var i = 0; i < lines.length; i++) {
    if (lines[i] != 0) {
      console.log('Case ' + (i+1) + ': ' + lines[i]);
    }
  }
});

File: introduction/ITP1_3_B-Print Test Cases.rs
use std::io::*;
fn main() {
  let stdin = stdin();
  for (i, line) in stdin.lock().lines().enumerate() {
    let x = line.unwrap();
    if x == "0" { break; }
    println!("Case {}: {}", i+1, x);
  }
}
